README.rst

```diff
@@ -37,6 +37,72 @@ Alternatively to get more up-to-date features:
 
 check the changelog_ for information about features included on `master` but not yet released.
 
+Dev Env Setup
+-------------
+
+Installation:
+    Python https://www.python.org/
+
+    Pyenv https://github.com/pyenv/pyenv#installation
+
+Python Version:
```

Comment:
hmmm rather than encoding this in the readme what do we think about just adding a `.python-version` file to enforce this a bit more specifically

Reply:
Ah. I can change this.